app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<header class="container d-flex justify-content-between  align-items-center my-3">
```

Comment:
When trying to create a new user a `no method error in Devise` error is shown. Please make the corresponding changes to fix them.

Check the image below for a better understanding: 
![Capture3](https://user-images.githubusercontent.com/75709904/149640532-41cc0608-10fb-4816-8334-38bb31ce1f82.PNG)